File: PythonActivities/Project_Activity8.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By

def Project_Activity8():
    driver = webdriver.Chrome()
    driver.get("https://alchemy.hguy.co/lms/")
    contactLink = driver.find_element(By.XPATH,"//a[@href='https://alchemy.hguy.co/lms/contact/']").click()

    fullNameTextBox = driver.find_element(By.XPATH,"//input[@id='wpforms-8-field_0']").send_keys('FST Test Team')
    emailTextBox = driver.find_element(By.XPATH,"//input[@id='wpforms-8-field_1']").send_keys('fsttestTeam@gmail.com')
    messaeTextBox = driver.find_element(By.XPATH,"//textarea[@id='wpforms-8-field_2']").send_keys('Great Courses. Thank you !!')
    sendMessageButton = driver.find_element(By.XPATH,"//button[@id='wpforms-submit-8']").click()
    returnMessage = driver.find_element(By.XPATH,"//div[@id='wpforms-confirmation-8']/p").text

    print(returnMessage)
Project_Activity8()

File: Selenium/Project/ProjectActivity8.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.testng.annotations.Test;

public class ProjectActivity8 extends BaseTest{
    @Test
    public void cotactTest(){
        WebElement contactLink = driver.findElement(By.xpath("//a[@href='https://alchemy.hguy.co/lms/contact/']"));
        contactLink.click();
        WebElement fullNameTxtbox = driver.findElement(By.xpath("//input[@id='wpforms-8-field_0']"));
        WebElement emailTxtbox = driver.findElement(By.xpath("//input[@id='wpforms-8-field_1']"));
        WebElement messageTextArea = driver.findElement(By.xpath("//textarea[@id='wpforms-8-field_2']"));
        WebElement sendMessageBtn = driver.findElement(By.xpath("//button[@class='wpforms-submit ']"));
        fullNameTxtbox.sendKeys("FST Test Team");
        emailTxtbox.sendKeys("fsttestTeam@gmail.com");
        messageTextArea.sendKeys("Great Courses. Thank you !!");
        sendMessageBtn.click();

        WebElement confirmationMessage = driver.findElement(By.xpath("//div[@class='wpforms-confirmation-container wpforms-confirmation-scroll']/p"));
        System.out.println("The message shown by the page is : "+confirmationMessage.getText());

    }
}
